examples/ssl_connection.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import txredisapi as redis
+
+from twisted.internet import defer, ssl
+from twisted.internet import reactor
+
+
+class TestContextFactory(ssl.ClientContextFactory):
+  def getContext(self):
+    ctx = ssl.ClientContextFactory.getContext(self)
+    # ctx.load_verify_locations('./test/ca.crt')
+    # ctx.use_certificate_file('./test/redis.crt')
+    # ctx.use_privatekey_file('./test/redis.key')
+    return ctx
+
+@defer.inlineCallbacks
+def main():
+    rc = yield redis.Connection(ssl_context_factory=TestContextFactory())
```

Comment:
There is no need for `yield` here since `Connection.__init__` can't be async

Reply:
I'm getting a `Deferred` back from that which than crashes 2 lines after due to `set` not existing in the the `Deferred` object.

Reply:
Wow, sorry, my fault. I forgot there are non-lazy variants of connection methods :laughing: